dbms/src/Storages/Page/Page.h

```diff
@@ -52,30 +52,36 @@ struct FieldOffsetInsidePage
 struct Page
 {
 public:
-    // only take the low u64, ignoring the high u64(NamespaceId)
-    explicit Page(const PageIdV3Internal & page_id_v3_)
-        : page_id(page_id_v3_.low)
+    static Page invalidPage()
     {
+        Page page;
+        page.page_id = INVALID_PAGE_U64_ID;
+        page.is_valid = false;
+        return page;
     }
 
     Page()
-        : page_id(INVALID_PAGE_ID)
+        : page_id(INVALID_PAGE_U64_ID)
+        , is_valid(true)
     {}
```

Comment:
Why we need a valid Page with `INVALID_PAGE_U64_ID`?

Reply:
OK, I will pass the page_id through the constructor.

Reply:
Fixed.